src/lid.c

```diff
@@ -89,61 +89,25 @@ enum LidLayerTypes {
     STOR,                    // storage layer
     PAVE,                    // pavement layer
     DRAINMAT,                // drainage mat layer
-    DRAIN};                  // underdrain system
+    DRAIN
+};                  // underdrain system
 
 //// Note: DRAINMAT must be placed before DRAIN so the two keywords can
 ///        be distinguished from one another when parsing a line of input. 
 
 char* LidLayerWords[] =
-    {"SURFACE", "SOIL", "STORAGE", "PAVEMENT", "DRAINMAT", "DRAIN", NULL};
+{ "SURFACE", "SOIL", "STORAGE", "PAVEMENT", "DRAINMAT", "DRAIN", NULL };
 
 char* LidTypeWords[] =
-    {"BC",                   //bio-retention cell
```

Comment:
I think the best way to do this would to be instead of moving these to `lid.h`, you should write it some simple hook functions here in `lid.c`.

Reply:
very nice work! @jennwuu and very good commenting @bemcdonnell If you can make an API for LID objects then further hydraulic connections should be simple(r)

Reply:
Sure! I can do that